gr-azure-software-radio/MANIFEST.md

```diff
@@ -0,0 +1,17 @@
+title: The azure_software_radio OOT Module
+brief: Short description of gr-azure_software_radio
```

Comment:
This is all still boilerplate. Can we update as appropriate?

Reply:
don't think we need this file, we describe the project in the main readme.md